README.md

```diff
@@ -2699,16 +2699,16 @@ Other Style Guides
   <a name="naming--leading-underscore"></a><a name="22.4"></a>
   - [22.4](#naming--leading-underscore) Do not use trailing or leading underscores. eslint: [`no-underscore-dangle`](http://eslint.org/docs/rules/no-underscore-dangle.html) jscs: [`disallowDanglingUnderscores`](http://jscs.info/rule/disallowDanglingUnderscores)
 
-    > Why? JavaScript does not have the concept of privacy in terms of properties or methods. Although a leading underscore is a common convention to mean “private”, in fact, these properties are fully public, and as such, are part of your public API contract. This convention might lead developers to wrongly think that a change won't count as breaking, or that tests aren't needed. tl;dr: if you want something to be “private”, it must not be observably present.
+    > Why? JavaScript does not have the concept of privacy in terms of properties or methods. Although a leading underscore is a common convention to mean "private", in fact, these properties are fully public, and as such, are part of your public API contract. This convention might lead developers to wrongly believe that a change will not count as breaking, or that tests are not needed. Tl; dr: if you want something to be "private", it should content with the prefix _.
```

Comment:
@MarcoBrunoBR eu mudaria o texto, porque ainda dá a ideia de que private com _ é ruim.

Sugestão:

This convention is not bad at all. Tests must be done, even on the private methods. Tl; dr: if you want something "private", it is ok to use prefix `_`.